### PR TITLE
Ref T39736, Update node version

### PR DESCRIPTION
**Pull request by luis30n**
Fix:
`docker-compose run (...) hex lerna run build`
is returning errors:
`error @lerna/add@4.0.0: The engine "node" is incompatible with this module. Expected version ">= 10.18.0". Got "10.15.3"`

### CODE REVIEW

File: packages/hex/yarn.lock [hunk 1]
```diff
@@ -2,13 +2,6 @@
 # yarn lockfile v1
 
 
-"@bitnami/hex-core@^4.0.0":
```
Comment (Angelmmiguel) [inline packages/hex/yarn.lock:5]: Is this change expected?

Comment (luis30n) [reply packages/hex/yarn.lock:5]: After running yarn dependecies it is added to the yarn.lock, should I remove this change?

Comment (Angelmmiguel) [reply packages/hex/yarn.lock:5]: Interesting. Well, let's keep the change :)

**Review by Angelmmiguel — APPROVED** [review]
LGTM

**Pull request merged**